src/Inkluzitron/Modules/Points/PointsModule.cs

```diff
@@ -15,19 +17,29 @@ namespace Inkluzitron.Modules.Points
     [Alias("points")]
     [Summary("Body se počítají stejně jako u GrillBot. Za každou reakci uživatel obdrží 0 až 10 bodů, za zprávu 0 až 25 bodů. Po odeslání zprávy " +
         "bot počítá jedno minutový cooldown. U reakce je cooldown 30 vteřin.")]
-    public class PointsModule : ModuleBase
+    public sealed class PointsModule : ModuleBase, IDisposable
     {
         private PointsService PointsService { get; }
+        private GraphPaintingService GraphPaintingService { get; }
         private DiscordSocketClient Client { get; }
         private ReactionSettings ReactionSettings { get; }
         private readonly int BoardPageLimit = 10;
 
-        public PointsModule(PointsService pointsService, DiscordSocketClient client,
-            ReactionSettings reactionSettings)
+        private PointsGraphPaintingStrategy GraphPaintingStrategy { get; }
+
+        public PointsModule(PointsService pointsService, DiscordSocketClient client, ReactionSettings reactionSettings, GraphPaintingService graphPaintingService)
         {
             PointsService = pointsService;
             Client = client;
             ReactionSettings = reactionSettings;
+            GraphPaintingService = graphPaintingService;
+            GraphPaintingStrategy = new PointsGraphPaintingStrategy();
+        }
+
+        public void Dispose()
```

Comment:
Pokud se to má volat po každým provedení příkazu, tak bych spíš použil override na AfterExecute.

Reply:
Dispose je fuč.